ibis-server/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "wren-core-wrapper"
```

Comment:
```suggestion
name = "wren-core-python"
```
Maybe `wren-core-python` better.

Reply:
I will rename the name to `wren-modeling-py` to pair with `wren-modeling-rs`.